gpjax/likelihoods.py

```diff
@@ -38,6 +37,10 @@ def link_function(self) -> Callable:
         """Return the link function of the likelihood function."""
         raise NotImplementedError
 
+    def init(self, key: jnp.ndarray) -> Dict:
+        """Initialise the stateful parameters of the likelihood function."""
+        return self.params
```

Comment:
What does this do?

(Calling this method results in an error, since `.params` is no longer there).